pkg/api/store/cluster/cluster_store.go

```diff
@@ -313,3 +320,19 @@ func validateNetworkFlag(data map[string]interface{}, create bool) error {
 
 	return nil
 }
+
+func enableLocalBackup(data map[string]interface{}) {
+	rkeConfig, ok := values.GetValue(data, "rancherKubernetesEngineConfig")
+
+	if ok && rkeConfig != nil {
+		backupConfig := values.GetValueN(data, "rancherKubernetesEngineConfig", "services", "etcd", "backupConfig")
+		if backupConfig == nil {
```

Comment:
should we also check the legacy snapshot value? And if not nil, and == true, don't configure the backup config